Position pos
string[] objects
float32[] objectsX
float32[] objectsY